src/overlays/actors/ovl_En_Demo_heishi/z_en_demo_heishi.h

```diff
@@ -8,11 +8,24 @@ struct EnDemoheishi;
 typedef void (*EnDemoheishiActionFunc)(struct EnDemoheishi*, GlobalContext*);
 
 typedef struct EnDemoheishi {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x110];
-    /* 0x0254 */ EnDemoheishiActionFunc actionFunc;
-    /* 0x0258 */ char unk_258[0x68];
-} EnDemoheishi; // size = 0x2C0
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable[17];
+    /* 0x1EE */ Vec3s morphTable[17];
+    /* 0x254 */ EnDemoheishiActionFunc actionFunc;
+    /* 0x258 */ s16 headRotY;
+    /* 0x25A */ s16 headRotX;
+    /* 0x25C */ s16 headRotZ;
+    /* 0x25E */ s16 headRotYTarget;
+    /* 0x260 */ s16 headRotXTarget;
+    /* 0x262 */ s16 headRotZTarget;
+    /* 0x264 */ s32 animIndex;
+    /* 0x268 */ s16 unk268;                         /* inferred */
+    /* 0x26C */ f32 frameCount;
+    /* 0x270 */ s16 unk270;                         /* inferred */
+    /* 0x272 */ s16 unk272;                         /* inferred */
+    /* 0x274 */ ColliderCylinder colliderCylinder;
+} EnDemoheishi;                                     /* size = 0x2C0 */
```

Comment:
```suggestion
} EnDemoheishi; // size = 0x2C0
```
Prefer to just keep the size comment the same as it was for consistency.

Reply:
Fixed